### PR TITLE
Reorganise locale files

### PR DESCRIPTION
**Pull request by Nitemaeric**
## Context

- The locale files nested structure doesn't match the locale key structure. This doesn't break anything, but it's strange.
- Service Name deserves to be outside of the scope of the application layout. We will need to use it elsewhere, such as in emails, notifications, references to the service name itself in all views. These will **need** to be consistent.

## Changes proposed in this pull request

- Reorganise locales to match locale key hierarchies
- Update `service_name` locale key to be view agnostic.

## Guidance to review

- Check that locale keys match their nested file structure.
  - E.g. The `en.claims.support.users.index.page_title` locale key should be hosted at `config/locales/en/claims/support/users.yml`.

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by Nitemaeric** [comment]
[Reorganise locale file hierarchies](https://trello.com/c/T2EBluH3/129-reorganise-locale-file-hierarchies)

**Review by CatalinVoineag — APPROVED** [review]
This looks good to me 👍🏻

**Comment by Nitemaeric** [comment]
@CatalinVoineag I've also updated the CODEOWNERS to match, so reviewers are added by default when we add locale changes :)

File: config/locales/en/placements.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+en:
+  placements:
+    service_name: Manage School Placements
```
Comment (JamieCleare2525) [inline config/locales/en/placements.yml:3]:
```suggestion
    service_name: Manage school placements
```

**Comment by CatalinVoineag** [comment]
> @CatalinVoineag I've also updated the CODEOWNERS to match, so reviewers are added by default when we add locale changes :)

Good idea 👍🏻

**Pull request merged**